safe_extras/parameters/numeric_parameter.py

```diff
@@ -9,9 +9,6 @@
 class NumericParameter(GenericParameter):
     """A subclass of generic parameter that accepts float or integer only.
 
-    Known issue : You should set the minimum and the maximum value explicitly.
-    See https://github.com/inasafe/inasafe/issues/2468#issuecomment-171124507
-
```

Comment:
Hmm, I did nothing regarding this issue. I think I will need to check the status of parameter in inasafe and the parameter repo itself.
After looking deeper, there are some difference between the two which shouldn't happen.